### DISCUSSION TITLE
mini-graph-card con valori a partire dalle 00:00

### DISCUSSION THREAD

**Discussion by vanzo95** [indomus/forum#1807] [HUB: Home Assistant]
Buonasera a tutti,

Vi chiedo cortesemente se c'è la possibilità di graficare con mini-graph-card in modo che lui prenda i valori a partire dalla mezzanotte e poi alla mezzanotte successiva si resetti.
C'è la possibilità di mostrare i grafici per le 24 ore precedenti, ma non a partire dalla mezzanotte.

Vi ringrazio!

**Comment by 50494554524F** [comment]
un paio di idee al volo, non sapendo che tipo di dati hai...
puoi provare a derivare un sensore con 
https://www.home-assistant.io/integrations/history_stats/
oppure creartene una copia e poi usare purge_entities alla mezzanotte per il sensore copia
oppure apexcharts / Plotly Graph Card

**Reply by vanzo95** [reply]
Grazie mille per la risposta! 
Avevo dei dati di un sensore di temperatura e volevo vedere i dati delle 24h così da poter visualizzare il minimo ed il massimo con il relativo orario.
Da quello che ho letto in giro mini-graph-card grafica le 24h a partire da "adesso" quindi non dalle 00:00. Un peccato.
Ho risolto il problema usando ApexCharts.
